assets/data/themes/drauven.json

```diff
@@ -276,13 +276,13 @@
 				{
 					"name": "Drauv_ArmL_2handed_lower",
 					"tint": "skin", "tintAmount": 0.4,
-					"depth": 1200,
+					"depth": 1215,
```

Comment:
🤔 Do you think this value be increased a bit more so that it's a higher number than the armour layers?

Reply:
Good idea, I'll set it to 1500